GooglePlayInstant/Editor/PlayInstantRunner.cs

```diff
@@ -0,0 +1,165 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using GooglePlayInstant.Editor.GooglePlayServices;
+using UnityEditor;
+using UnityEngine;
+#if UNITY_2018_1_OR_NEWER
+using UnityEditor.Build.Reporting;
+#endif
+
+namespace GooglePlayInstant.Editor
+{
+    public static class PlayInstantRunner
+    {
+        private const string BuildAndRunErrorTitle = "Build and Run Error";
+        private const string OkButtonText = "OK";
+        private const string CancelButtonText = "Cancel";
+
+        public static void BuildAndRun()
+        {
+            if (!PlayInstantBuildConfiguration.IsPlayInstantScriptingSymbolDefined())
+            {
+                Debug.LogError("Build and Run halted since selected platform is Installed");
+                const string message = "The currently selected Android Platform is \"Installed\".\n\n" +
+                                       "Click \"OK\" to open the \"Configure Instant or Installed\" " +
+                                       "window where the platform can be changed to \"Instant\".";
+                if (EditorUtility.DisplayDialog(BuildAndRunErrorTitle, message, OkButtonText, CancelButtonText))
+                {
+                    PlayInstantSettingsWindow.ShowWindow();
+                }
+
+                return;
+            }
+
+            var jarPath = Path.Combine(AndroidSdkManager.AndroidSdkRoot, "extras/google/instantapps/tools/ia.jar");
```

Comment:
nit: the ia.jar path would be good as a constant

Reply:
Done.